website_certificate/certificate_view.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record id="view_website_config_settings" model="ir.ui.view">
+            <field name="model">website.config.settings</field>
+            <field name="name">website.config.settings.form.inherit</field>
+            <field name="inherit_id" ref="website.view_website_config_settings"/>
+            <field name="arch" type="xml">
+                <div name="social_googleplus" position="after">
+                    <label for="certificate_number1"/>
+                    <div class="oe_inline">
+                        <field name="certificate_number1" placeholder="沪ICP备12005091号"/>
```

Comment:
this is originally for the ICP. I think it doesnot hurt (second one could be for Comodo example)